src/components/LocationInput/LocationInput.jsx

```diff
@@ -96,7 +95,6 @@ LocationInput.propTypes = {
       longitude: PropTypes.number.isRequired,
     }),
   }),
-  // ({address:str,coordinates:{longitude:float,latitude:float}}) -> void
```

Comment:
unfortunately, there is no way to specify the signature of a callable with proptypes. with typescript we can specify this natively, but we javascript we can't, and proptypes doesn't solve it (there's [this feature request from 2015](https://github.com/facebook/react/issues/4811), but it was never added. and now proptypes is unmaintained afaik)

that's why I added this comment. without it, we know that the property has to be a callable, but we don't know the signature that that callable should have. how many arguments should it expect? and of what type? this inline comment at least captures that. even if it's pseodocode, I think it's valuable because it clearly shows what type of argument the callback should receive, what it should look like. it also captures the fact that the callback is not expected to return anything (-> void)

unless you have a strong objection, please add this comment back